src/Terminal/ui/TerminalInput.tsx

```diff
@@ -44,6 +53,8 @@ export function TerminalInput(): React.ReactElement {
   const [value, setValue] = useState(command);
   const [postUpdateValue, setPostUpdateValue] = useState<{ postUpdate: () => void } | null>();
   const [possibilities, setPossibilities] = useState<string[]>([]);
+  const [searchResults, setSearchResults] = useState<string[]>([]);
+  const [autofilledValue, setAutofilledValue] = useState<boolean>(false);
```

Comment:
Just as a note: it's optional  to type `useState` hooks if they have a default value that can be inferred